locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Community Committee
+
+The Community Committee (CommComm) is a top-level committee in the Node.js Foundation focused on community-facing efforts. Code commits !== the only way to contribute!
+
+The CommComm has authority over community outreach including:
+ - Community Evangelism
+ - Education Initiatives
+ - Cultural Direction of Node.js Foundation
+ - Community Organization Outreach
+ - Project Moderation/Mediation
+ - Public Outreach and [Publications](https://medium.com/the-node-js-collection)
+
+There are four levels of engagement with the Community Committee:
+
+ 1. A **Contributor** is any individual creating or commenting on an issue or pull request.
+ 2. A **Collaborator** is a contributor who has been given write access to the repository
+ 3. An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+ 4. A **Member** is a collaborator with voting rights who has met the requirements of participation and voted in by the CommComm voting process.
+
+For the current list of Community Committee members, see the project's [README.md](https://github.com/nodejs/community-committee).
+
+## Collaborators
+
+It is the mission of CommComm to further build out the Node.js Community. If you're reading this, you're already a part of that community – and as a part of the Node.js Community, we'd love to have your help!
+
+The [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub repository is a great place to start. Check out the [issues labeled "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue) to see where we're looking for help. If you have your own ideas on how we could engage and build the community, feel free to create your own issue.
+
+If you're interested in participating in the Community Committee as a member, you should create an issue asking to be an Observer in our next Community Committee meeting. You can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142).
+
+Before diving in though, please be sure to read the [Collaborator Guide](https://github.com/nodejs/community-committee/blob/master/COLLABORATOR_GUIDE.md).
+
+## Membership
+
+To become a member of the Community Committee, start participating! You must:
+
+ 1. Attend the bi-weekly meetings, investigate issues tagged as good first issue, file issues and pull requests, and provide insight via GitHub.
```

Comment:
I started rewriting this one bullet, but it quickly grew beyond what a single bullet should be 😬 

I'd like to suggest a full rework of this section - feel free to modify or suggest any changes. 

```
- Attend the bi-weekly CommComm meetings - you can request the bi-weekly meetings by requesting to become an observer (see an example of such a request [here](https://github.com/nodejs/community-committee/issues/142)). Once added as an Observer to meetings, we will track attendance and participation for 3 months as a part of becoming a membership, as defined by the CommComm Governance.
- Take on introductory issues labeled as "Good First Issue", file issues with your ideas for what the CommComm can work on, create  pull requests with improvements to our existing work, and help us by sharing your thoughts and ideas in the ongoing discussions we're having in GitHub.
- Participate in our ongoing efforts around community building - like localization, evangelism, the Node.js Collection, and others - by digging into their respective repositories and getting involved!
```